_includes/navigation.html

```diff
@@ -1,10 +1,12 @@
-<nav class="navigation">
-  <ul>
-    <li>
-      <a href="http://github.com/thoughtbot/ember-cli-rails">
-        <img class="navigation__icon" src="images/github-icon.svg" alt="github ember-cli-rails repository">
-        GitHub
-      </a>
-    </li>
-  </ul>
-</nav>
+<header>
```

Comment:
I would have the whole top of the page be the header. ¯\_(ツ)_/¯

> The HTML <header> element represents a group of introductory or navigational aids. It may contain some heading elements but also other elements like a logo, wrapped section's header, a search form, and so on.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header